From b1e93471d0ac6a0662bc18b63d0aaa67ac519bfb Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Mon, 27 Jan 2025 15:07:59 -0600
Subject: [PATCH 2/2] fixup

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b8e5c6591086..f80e78c7cf20 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 6
 PATCHLEVEL = 6
-SUBLEVEL = 74
+SUBLEVEL = 58
 EXTRAVERSION =
 NAME = Pinguïn Aangedreven
 
-- 
2.45.2

