From ff92589368541b8c5263c827f2c0f0c040390634 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 25 Aug 2015 14:40:55 -0500
Subject: [PATCH] omapdrm: omap_crtc_atomic_set_property dont EINVAL

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/gpu/drm/omapdrm/omap_crtc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/omapdrm/omap_crtc.c b/drivers/gpu/drm/omapdrm/omap_crtc.c
index 0ad9a94..90a0136 100644
--- a/drivers/gpu/drm/omapdrm/omap_crtc.c
+++ b/drivers/gpu/drm/omapdrm/omap_crtc.c
@@ -552,8 +552,6 @@ static int omap_crtc_atomic_set_property(struct drm_crtc *crtc,
 		omap_state->default_color = val;
 	else if (property == priv->alpha_blender_prop)
 		omap_state->partial_alpha_enabled = !!val;
-	else
-		return -EINVAL;
 
 	return 0;
 }
-- 
2.5.0

