From 06f499122426eaf2d0149b84ebe51868fbe8f368 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 4 Nov 2015 16:06:18 -0600
Subject: [PATCH] spi: disable dma, fixes spidev loaded as an overlay

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/spi/spi-omap2-mcspi.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/spi/spi-omap2-mcspi.c b/drivers/spi/spi-omap2-mcspi.c
index b2165db..2b3dd7b 100644
--- a/drivers/spi/spi-omap2-mcspi.c
+++ b/drivers/spi/spi-omap2-mcspi.c
@@ -955,11 +955,11 @@ static int omap2_mcspi_request_dma(struct spi_device *spi)
 	dma_cap_set(DMA_SLAVE, mask);
 	sig = mcspi_dma->dma_rx_sync_dev;
 
-	mcspi_dma->dma_rx =
-		dma_request_slave_channel_compat(mask, omap_dma_filter_fn,
-						 &sig, &master->dev,
-						 mcspi_dma->dma_rx_ch_name);
-	if (!mcspi_dma->dma_rx)
+	//mcspi_dma->dma_rx =
+	//	dma_request_slave_channel_compat(mask, omap_dma_filter_fn,
+	//					 &sig, &master->dev,
+	//					 mcspi_dma->dma_rx_ch_name);
+	//if (!mcspi_dma->dma_rx)
 		goto no_dma;
 
 	sig = mcspi_dma->dma_tx_sync_dev;
-- 
2.6.2

