From 4ef2bb02d670bd5b435bfd90b287ff8961a7e9fb Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 17 May 2016 12:51:10 -0500
Subject: [PATCH 1/4] fix sleep33xx.S: for thumb2

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 arch/arm/mach-omap2/sleep33xx.S | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-omap2/sleep33xx.S b/arch/arm/mach-omap2/sleep33xx.S
index 8f3186e..1631ba0 100644
--- a/arch/arm/mach-omap2/sleep33xx.S
+++ b/arch/arm/mach-omap2/sleep33xx.S
@@ -30,6 +30,7 @@
 
 	.text
 	.align 3
+	.arm
 
 ENTRY(am33xx_do_wfi)
 	stmfd	sp!, {r4 - r11, lr}	@ save registers on stack
-- 
2.8.1

