From 7dbe111f26315faf574f8c1b0410ef31187da770 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 17 May 2016 12:56:30 -0500
Subject: [PATCH 3/4] fix ti-emif-sram-pm.S: for thumb2

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/memory/ti-emif-sram-pm.S | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/memory/ti-emif-sram-pm.S b/drivers/memory/ti-emif-sram-pm.S
index 63f6201..e45096b 100644
--- a/drivers/memory/ti-emif-sram-pm.S
+++ b/drivers/memory/ti-emif-sram-pm.S
@@ -34,6 +34,7 @@
 
 	.text
 	.align 3
+	.arm
 
 ENTRY(ti_emif_sram)
 
-- 
2.8.1

