From b3306258eb7ccdae4200dd7f57e9dd033008cfa8 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 4 Aug 2016 14:42:20 -0500
Subject: [PATCH 5/5] fix: 4.1.29+rt, big opps

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 mm/swap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/swap.c b/mm/swap.c
index e972abe..105bb49 100644
--- a/mm/swap.c
+++ b/mm/swap.c
@@ -636,7 +636,7 @@ static void __lru_cache_add(struct page *page)
 	struct pagevec *pvec = &get_locked_var(swapvec_lock, lru_add_pvec);
 
 	page_cache_get(page);
-	if (!pagevec_add(pvec, page) || PageCompound(page))
+	if (!pagevec_space(pvec) || PageCompound(page))
 		__pagevec_lru_add(pvec);
 	pagevec_add(pvec, page);
 	put_locked_var(swapvec_lock, lru_add_pvec);
-- 
2.8.1

