From 4a7106ae982e3ae57d520b44c614235a57acacb8 Mon Sep 17 00:00:00 2001
From: Matt Ranostay <matt.ranostay@konsulko.com>
Date: Mon, 30 Aug 2021 23:07:55 +0300
Subject: [PATCH 4/4] hwmon: tmp468: add device tree support

Signed-off-by: Matt Ranostay <matt.ranostay@konsulko.com>
---
 drivers/hwmon/tmp468.c | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/drivers/hwmon/tmp468.c b/drivers/hwmon/tmp468.c
index 6faaa4694591..ab55567b2896 100644
--- a/drivers/hwmon/tmp468.c
+++ b/drivers/hwmon/tmp468.c
@@ -82,6 +82,19 @@ static const struct i2c_device_id tmp468_id[] = {
 };
 MODULE_DEVICE_TABLE(i2c, tmp468_id);
 
+static const struct of_device_id tmp468_of_match[] = {
+	{
+		.compatible = "ti,tmp464",
+		.data = (void *)tmp464,
+	},
+	{
+		.compatible = "ti,tmp468",
+		.data = (void *)tmp468,
+	},
+	{},
+};
+MODULE_DEVICE_TABLE(of, tmp468_of_match);
+
 /*
  * Client data
  */
@@ -488,6 +501,7 @@ static struct i2c_driver tmp468_driver = {
 	.class		= I2C_CLASS_HWMON,
 	.driver = {
 		.name	= "tmp468",
+		.of_match_table = of_match_ptr(tmp468_of_match),
 	},
 	.probe		= tmp468_probe,
 	.id_table	= tmp468_id,
-- 
2.30.2

