From 3da0ada1606b94f83caa35402fce96de09c9cb19 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Fri, 9 Mar 2018 12:55:11 -0600
Subject: [PATCH] ti undo overlays

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 arch/arm/boot/dts/Makefile | 30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 4fc43ce05673..3a8968d045aa 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -712,9 +712,9 @@ dtb-merge-$(CONFIG_SOC_DRA7XX) += \
 	am57xx-evm-reva3.dtb \
 	am57xx-evm-reva3-cam-mt9t111.dtb \
 	am57xx-evm-reva3-cam-ov10635.dtb
-ifeq ($(CONFIG_SOC_DRA7XX),y)
-dts-dirs += ti
-endif
+#ifeq ($(CONFIG_SOC_DRA7XX),y)
+#dts-dirs += ti
+#endif
 dtb-$(CONFIG_ARCH_ORION5X) += \
 	orion5x-kuroboxpro.dtb \
 	orion5x-lacie-d2-network.dtb \
@@ -1117,24 +1117,24 @@ dtb-$(CONFIG_ARCH_ASPEED) += aspeed-bmc-opp-palmetto.dtb \
 	aspeed-ast2500-evb.dtb
 endif
 
-ifeq ($(BUILD_ITBS),y)
-dtb-y=
-dtb-merge-y=
-endif
+#ifeq ($(BUILD_ITBS),y)
+#dtb-y=
+#dtb-merge-y=
+#endif
 
 dtstree		:= $(srctree)/$(src)
 dtb-$(CONFIG_OF_ALL_DTBS) := $(patsubst $(dtstree)/%.dts,%.dtb, $(wildcard $(dtstree)/*.dts))
 
-always		:= $(dtb-y) $(dtb-merge-y)
+always		:= $(dtb-y)
 clean-files	:= *.dtb
 
-$(addprefix $(obj)/,$(dtb-merge-y)): TI_DTBOS
-	@$(srctree)/scripts/dtb-merge $(srctree) $(objtree) $@ $(objtree)/scripts/dtc/fdtoverlay $(src)/ti
+#$(addprefix $(obj)/,$(dtb-merge-y)): TI_DTBOS
+#	@$(srctree)/scripts/dtb-merge $(srctree) $(objtree) $@ $(objtree)/scripts/dtc/fdtoverlay $(src)/ti
 
-TI_DTBOS:
-	$(Q)$(MAKE) $(build)=$(src)/ti
+#TI_DTBOS:
+#	$(Q)$(MAKE) $(build)=$(src)/ti
 
-PHONY += TI_DTBOS
+#PHONY += TI_DTBOS
 
-subdir-y	+= $(dts-dirs)
-subdir-		+= ti
+#subdir-y	+= $(dts-dirs)
+#subdir-		+= ti
-- 
2.16.1

