From f330e90a9a50c8f6d72b2fc79e90ed00463769b9 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 16 Mar 2021 16:53:01 -0500
Subject: [PATCH] xenomai fixes

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 init/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/Kconfig b/init/Kconfig
index b6bb65b8c301..d1c1622ea666 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -2053,7 +2053,7 @@ menuconfig XENOMAI
 	  Please visit http://xenomai.org for more information.
 
 if XENOMAI
-source "arch/$SRCARCH/xenomai/Kconfig"
+source "arch/arm/xenomai/Kconfig"
 endif
 
 if MIGRATION
-- 
2.30.2

