From eaf21b72c65e260025b250407f414c420f1e96f0 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 8 Sep 2016 11:52:24 -0500
Subject: [PATCH 15/15] ti/cpsw: priv -> cpsw (slaves)

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/net/ethernet/ti/cpsw.c | 42 ++++++++++++++++++++++++------------------
 1 file changed, 24 insertions(+), 18 deletions(-)

diff --git a/drivers/net/ethernet/ti/cpsw.c b/drivers/net/ethernet/ti/cpsw.c
index ebcde6c..235ffbf 100644
--- a/drivers/net/ethernet/ti/cpsw.c
+++ b/drivers/net/ethernet/ti/cpsw.c
@@ -2243,10 +2243,11 @@ static int cpsw_phy_read_reg(struct net_device *ndev,
 			     struct ethtool_phy_reg *data)
 {
 	struct cpsw_priv *priv = netdev_priv(ndev);
-	int slave_no = cpsw_slave_index(priv);
+	struct cpsw_common *cpsw = priv->cpsw;
+	int slave_no = cpsw_slave_index(cpsw, priv);
 
-	if (priv->slaves[slave_no].phy)
-		phy_ethtool_read_reg(priv->slaves[slave_no].phy, data);
+	if (cpsw->slaves[slave_no].phy)
+		phy_ethtool_read_reg(cpsw->slaves[slave_no].phy, data);
 	return 0;
 }
 
@@ -2254,10 +2255,11 @@ static int cpsw_phy_write_reg(struct net_device *ndev,
 			      struct ethtool_phy_reg *data)
 {
 	struct cpsw_priv *priv = netdev_priv(ndev);
-	int slave_no = cpsw_slave_index(priv);
+	struct cpsw_common *cpsw = priv->cpsw;
+	int slave_no = cpsw_slave_index(cpsw, priv);
 
-	if (priv->slaves[slave_no].phy)
-		phy_ethtool_write_reg(priv->slaves[slave_no].phy, data);
+	if (cpsw->slaves[slave_no].phy)
+		phy_ethtool_write_reg(cpsw->slaves[slave_no].phy, data);
 	return 0;
 }
 
@@ -2265,10 +2267,11 @@ static int cpsw_phy_mac_if_set(struct net_device *ndev,
 			       struct ethtool_phy_cmd *data)
 {
 	struct cpsw_priv *priv = netdev_priv(ndev);
-	int slave_no = cpsw_slave_index(priv);
+	struct cpsw_common *cpsw = priv->cpsw;
+	int slave_no = cpsw_slave_index(cpsw, priv);
 
-	if (priv->slaves[slave_no].phy)
-		phy_ethtool_mac_if_set(priv->slaves[slave_no].phy, data);
+	if (cpsw->slaves[slave_no].phy)
+		phy_ethtool_mac_if_set(cpsw->slaves[slave_no].phy, data);
 	return 0;
 }
 
@@ -2276,30 +2279,33 @@ static int cpsw_phy_mac_if_get(struct net_device *ndev,
 			       struct ethtool_phy_cmd *data)
 {
 	struct cpsw_priv *priv = netdev_priv(ndev);
-	int slave_no = cpsw_slave_index(priv);
+	struct cpsw_common *cpsw = priv->cpsw;
+	int slave_no = cpsw_slave_index(cpsw, priv);
 
-	if (priv->slaves[slave_no].phy)
-		phy_ethtool_mac_if_get(priv->slaves[slave_no].phy, data);
+	if (cpsw->slaves[slave_no].phy)
+		phy_ethtool_mac_if_get(cpsw->slaves[slave_no].phy, data);
 	return 0;
 }
 
 static int cpsw_phy_edge_rate_set(struct net_device *ndev, u8 *rate)
 {
 	struct cpsw_priv *priv = netdev_priv(ndev);
-	int slave_no = cpsw_slave_index(priv);
+	struct cpsw_common *cpsw = priv->cpsw;
+	int slave_no = cpsw_slave_index(cpsw, priv);
 
-	if (priv->slaves[slave_no].phy)
-		phy_ethtool_edge_rate_set(priv->slaves[slave_no].phy, rate);
+	if (cpsw->slaves[slave_no].phy)
+		phy_ethtool_edge_rate_set(cpsw->slaves[slave_no].phy, rate);
 	return 0;
 }
 
 static int cpsw_phy_edge_rate_get(struct net_device *ndev, u8 *rate)
 {
 	struct cpsw_priv *priv = netdev_priv(ndev);
-	int slave_no = cpsw_slave_index(priv);
+	struct cpsw_common *cpsw = priv->cpsw;
+	int slave_no = cpsw_slave_index(cpsw, priv);
 
-	if (priv->slaves[slave_no].phy)
-		phy_ethtool_edge_rate_get(priv->slaves[slave_no].phy, rate);
+	if (cpsw->slaves[slave_no].phy)
+		phy_ethtool_edge_rate_get(cpsw->slaves[slave_no].phy, rate);
 	return 0;
 }
 
-- 
2.9.3

