From dcd74eb72882f211f9cdaca1cf8992617eb61c8c Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 23 Jan 2025 14:54:59 -0600
Subject: [PATCH] sdhci-omap: disable eMMC/SD suspend at idle

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/mmc/host/sdhci-omap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/host/sdhci-omap.c b/drivers/mmc/host/sdhci-omap.c
index 5841a9afeb9f..f733f21c00aa 100644
--- a/drivers/mmc/host/sdhci-omap.c
+++ b/drivers/mmc/host/sdhci-omap.c
@@ -1340,7 +1340,7 @@ static int sdhci_omap_probe(struct platform_device *pdev)
 	mmc->caps |= MMC_CAP_NEED_RSP_BUSY;
 
 	/* Allow card power off and runtime PM for eMMC/SD card devices */
-	mmc->caps |= MMC_CAP_POWER_OFF_CARD | MMC_CAP_AGGRESSIVE_PM;
+	//mmc->caps |= MMC_CAP_POWER_OFF_CARD | MMC_CAP_AGGRESSIVE_PM;
 
 	ret = sdhci_setup_host(host);
 	if (ret)
-- 
2.45.2

