From ae35c604ec8fc1ce9f2fa23cbf524d7aeafbfe00 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 12 Jul 2022 13:49:50 -0500
Subject: [PATCH 3/3] uio/uio_pruss.c: ARCH_K3 try UIO_MEM_IOVA

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/uio/uio_pruss.c | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/drivers/uio/uio_pruss.c b/drivers/uio/uio_pruss.c
index dabc37ed6459..e600a8873793 100644
--- a/drivers/uio/uio_pruss.c
+++ b/drivers/uio/uio_pruss.c
@@ -224,7 +224,11 @@ static int pruss_probe(struct platform_device *pdev)
 		p->mem[0].name = "pruss";
 		p->mem[0].addr = regs_prussio->start;
 		p->mem[0].size = resource_size(regs_prussio);
+#ifdef CONFIG_ARCH_K3
+		p->mem[0].memtype = UIO_MEM_IOVA;
+#else
 		p->mem[0].memtype = UIO_MEM_PHYS;
+#endif
 
 		/* note: some userspace code uses hardcoded mem indices... */
 #ifdef CONFIG_ARCH_DAVINCI_DA850
@@ -241,6 +245,11 @@ static int pruss_probe(struct platform_device *pdev)
 		p->mem[1].name = "ddr";
 		p->mem[1].addr = gdev->ddr_paddr;
 		p->mem[1].size = extram_pool_sz;
+#ifdef CONFIG_ARCH_K3
+		p->mem[1].memtype = UIO_MEM_IOVA;
+#else
+		p->mem[1].memtype = UIO_MEM_PHYS;
+#endif
 		p->mem[1].memtype = UIO_MEM_PHYS;
 #endif
 
-- 
2.30.2

