From 1c29dca48eae65726f3f23f5941b2a491117e7a5 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 20 Oct 2015 12:05:37 -0500
Subject: [PATCH 4/4] quiet: arch/arm/mach-omap2/voltage.c: legacy harmless

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 arch/arm/mach-omap2/voltage.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-omap2/voltage.c b/arch/arm/mach-omap2/voltage.c
index cba8cad..81cf6eb 100644
--- a/arch/arm/mach-omap2/voltage.c
+++ b/arch/arm/mach-omap2/voltage.c
@@ -235,7 +235,7 @@ int __init omap_voltage_late_init(void)
 	struct voltagedomain *voltdm;
 
 	if (list_empty(&voltdm_list)) {
-		pr_err("%s: Voltage driver support not added\n",
+		pr_info("%s: Voltage driver support not added\n",
 			__func__);
 		return -EINVAL;
 	}
-- 
2.6.1

