From f317555e1e785ddce590fa1108bed54082777802 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 22 Jul 2021 18:22:28 -0500
Subject: [PATCH 3/3] builddeb: copy arm64/boot/Image to /boot/

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 scripts/package/builddeb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index f86c8c9f9452..48a2ac1d0917 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -148,6 +148,7 @@ else
 	cp $KCONFIG_CONFIG "$tmpdir/boot/config-$version"
 fi
 cp "$($MAKE -s -f $srctree/Makefile image_name)" "$tmpdir/$installed_image_path"
+cp arch/arm64/boot/Image "$tmpdir/boot/Image-$version"
 
 if is_enabled CONFIG_OF_EARLY_FLATTREE; then
 	# Only some architectures with OF support have this target
-- 
2.30.2

