From dc19947185ffeab0a875c1858a7e6d7f5627a2ee Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 5 May 2026 18:43:25 -0500
Subject: [PATCH] lzo: undo lzorle1x_1_compress_safe

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 crypto/lzo-rle.c | 2 +-
 crypto/lzo.c     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/crypto/lzo-rle.c b/crypto/lzo-rle.c
index 0abc2d87f042..0631d975bfac 100644
--- a/crypto/lzo-rle.c
+++ b/crypto/lzo-rle.c
@@ -55,7 +55,7 @@ static int __lzorle_compress(const u8 *src, unsigned int slen,
 	size_t tmp_len = *dlen; /* size_t(ulong) <-> uint on 64 bit */
 	int err;
 
-	err = lzorle1x_1_compress_safe(src, slen, dst, &tmp_len, ctx);
+	err = lzorle1x_1_compress(src, slen, dst, &tmp_len, ctx);
 
 	if (err != LZO_E_OK)
 		return -EINVAL;
diff --git a/crypto/lzo.c b/crypto/lzo.c
index 8338851c7406..ebda132dd22b 100644
--- a/crypto/lzo.c
+++ b/crypto/lzo.c
@@ -55,7 +55,7 @@ static int __lzo_compress(const u8 *src, unsigned int slen,
 	size_t tmp_len = *dlen; /* size_t(ulong) <-> uint on 64 bit */
 	int err;
 
-	err = lzo1x_1_compress_safe(src, slen, dst, &tmp_len, ctx);
+	err = lzo1x_1_compress(src, slen, dst, &tmp_len, ctx);
 
 	if (err != LZO_E_OK)
 		return -EINVAL;
-- 
2.53.0

