From b30cc974747e472c3f1cd11d5b00daf1a2a02da9 Mon Sep 17 00:00:00 2001
From: Jason Kridner <jkridner@beagleboard.org>
Date: Tue, 4 Jan 2022 22:00:47 -0500
Subject: [PATCH 14/41] greybus: Kconfig syntax update

Signed-off-by: Jason Kridner <jkridner@beagleboard.org>
---
 drivers/greybus/Kconfig | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/drivers/greybus/Kconfig b/drivers/greybus/Kconfig
index e03c3c842b69..6ffdab025c85 100644
--- a/drivers/greybus/Kconfig
+++ b/drivers/greybus/Kconfig
@@ -30,10 +30,11 @@ config GREYBUS_ES2
 
 config GREYBUS_NETLINK
 	tristate "Greybus netlink host controller"
-	---help---
-	Select this option if you want to implement a Greybus
-	"host controller" in userspace.
-	To compile this code as a module, chose M here: the module
-	will be called gb-netlink.ko
+	help
+	  Select this option if you want to implement a Greybus
+	  "host controller" in userspace.
+	  To compile this code as a module, chose M here: the module
+	  will be called gb-netlink.ko
+
 endif	# GREYBUS
 
-- 
2.30.2

