From 599196969eee54b2f486a1b72d9e181f5a3dcf0f Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 28 Sep 2021 11:33:53 -0500
Subject: [PATCH 2/2] hwmon/sht3x.c: dt support

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/hwmon/sht3x.c | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/drivers/hwmon/sht3x.c b/drivers/hwmon/sht3x.c
index 3f279aa1cee5..60f8971a16f5 100644
--- a/drivers/hwmon/sht3x.c
+++ b/drivers/hwmon/sht3x.c
@@ -738,9 +738,16 @@ static const struct i2c_device_id sht3x_ids[] = {
 	{"sts3x", sts3x},
 	{}
 };
-
 MODULE_DEVICE_TABLE(i2c, sht3x_ids);
 
+static const struct of_device_id shtc1_of_match[] = {
+	{ .compatible = "sensirion,shtc1" },
+	{ .compatible = "sensirion,shtw1" },
+	{ .compatible = "sensirion,shtc3" },
+	{ }
+};
+MODULE_DEVICE_TABLE(of, shtc1_of_match);
+
 static struct i2c_driver sht3x_i2c_driver = {
 	.driver.name = "sht3x",
 	.probe_new   = sht3x_probe,
-- 
2.30.2

