From 09c9f3497e5f49009f699dcc69b5a766970aadff Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 11 Oct 2016 10:53:48 -0500
Subject: [PATCH 4/4] fix: tieqep.c

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/misc/tieqep.c         | 2 +-
 2 files changed, 7 insertions(+), 4 deletions(-)

diff --git a/drivers/misc/tieqep.c b/drivers/misc/tieqep.c
index a0652f9..4d3e79b 100644
--- a/drivers/misc/tieqep.c
+++ b/drivers/misc/tieqep.c
@@ -489,7 +489,7 @@ static int eqep_probe(struct platform_device *pdev)
 	}
 
 	/* Get a handle to the system clock object */
-	clk = devm_clk_get(&pdev->dev, "fck");
+	clk = devm_clk_get(pdev->dev.parent, "fck");
 	if (IS_ERR(clk)) {
 		dev_err(&pdev->dev, "failed to get clock\n");
 		return PTR_ERR(clk);
-- 
2.9.3

