From 558a61815867931aba1376724537bdd843baaf2a Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 11 Jul 2019 22:08:14 -0500
Subject: [PATCH] sound: give us SND_SOC_PCM5102A

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 sound/soc/ti/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/ti/Kconfig b/sound/soc/ti/Kconfig
index 41342823e662..37b94b7f0287 100644
--- a/sound/soc/ti/Kconfig
+++ b/sound/soc/ti/Kconfig
@@ -28,6 +28,7 @@ config SND_SOC_DAVINCI_MCASP
 	select SND_SOC_TI_EDMA_PCM if TI_EDMA
 	select SND_SOC_TI_SDMA_PCM if DMA_OMAP
 	select SND_SOC_TI_UDMA_PCM if TI_K3_UDMA
+	select SND_SOC_PCM5102A
 	help
 	  Say Y or M here if you want to have support for McASP IP found in
 	  various Texas Instruments SoCs like:
-- 
2.20.1

