From 66c17e49d91d9e3b507981927322d3b2bb24bc28 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 22 Apr 2020 11:06:16 -0500
Subject: [PATCH] sound: give us SND_SOC_PCM5102A

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 sound/soc/ti/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/ti/Kconfig b/sound/soc/ti/Kconfig
index 278f6f8bb2b2..914fd701ad3c 100644
--- a/sound/soc/ti/Kconfig
+++ b/sound/soc/ti/Kconfig
@@ -29,6 +29,7 @@ config SND_SOC_DAVINCI_MCASP
 	select SND_SOC_TI_EDMA_PCM
 	select SND_SOC_TI_SDMA_PCM
 	select SND_SOC_TI_UDMA_PCM
+	select SND_SOC_PCM5102A
 	help
 	  Say Y or M here if you want to have support for McASP IP found in
 	  various Texas Instruments SoCs like:
-- 
2.26.1

