From d6c65abac39d8fdb0cbeb0365d3ae43ec6096354 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 13 Nov 2018 11:07:46 -0600
Subject: [PATCH] sound: give us SND_SOC_PCM5102A

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 sound/soc/davinci/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/davinci/Kconfig b/sound/soc/davinci/Kconfig
index cf686dbf986b..3451d6ea528a 100644
--- a/sound/soc/davinci/Kconfig
+++ b/sound/soc/davinci/Kconfig
@@ -44,6 +44,7 @@ config SND_AM33XX_SOC_EVM
 	tristate "SoC Audio for the AM33XX chip based boards"
 	depends on SND_EDMA_SOC && SOC_AM33XX && I2C
 	select SND_DAVINCI_SOC_GENERIC_EVM
+	select SND_SOC_PCM5102A
 	help
 	  Say Y or M if you want to add support for SoC audio on AM33XX
 	  boards using McASP and TLV320AIC3X codec. For example AM335X-EVM,
-- 
2.19.1

