From fb5de6537a927b6fb2dfeafed728d2860f5528fc Mon Sep 17 00:00:00 2001
From: Jeremy Stashluk <stashlukj@geophysical.com>
Date: Fri, 7 Sep 2018 10:35:59 -0400
Subject: [PATCH 3/4] dra7: add ti pwmss eqep
To: bugs@rcn-ee.com

---
 arch/arm/boot/dts/dra7.dtsi | 27 +++++++++++++++++++++++++++
 1 file changed, 27 insertions(+)

diff --git a/arch/arm/boot/dts/dra7.dtsi b/arch/arm/boot/dts/dra7.dtsi
index ecddb8b..9df890b 100644
--- a/arch/arm/boot/dts/dra7.dtsi
+++ b/arch/arm/boot/dts/dra7.dtsi
@@ -2165,6 +2165,15 @@
 				status = "disabled";
 			};
 
+			eqep0: eqep@0x4843e180 {
+				compatible = "ti,am33xx-eqep";
+				reg = <0x4843e180 0x80>;
+				clocks = <&l4_root_clk_div>;
+				clock-names = "fck";
+				interrupt-parent = <&crossbar_mpu>;
+				interrupts = <GIC_SPI 210 IRQ_TYPE_LEVEL_HIGH>;
+			};
+
 			ecap0: ecap@4843e100 {
 				compatible = "ti,dra746-ecap",
 					     "ti,am3352-ecap";
@@ -2195,6 +2204,15 @@
 				status = "disabled";
 			};
 
+			eqep1: eqep@0x48440180 {
+				compatible = "ti,am33xx-eqep";
+				reg = <0x48440180 0x80>;
+				clocks = <&l4_root_clk_div>;
+				clock-names = "fck";
+				interrupt-parent = <&crossbar_mpu>;
+				interrupts = <GIC_SPI 211 IRQ_TYPE_LEVEL_HIGH>;
+			};
+
 			ecap1: ecap@48440100 {
 				compatible = "ti,dra746-ecap",
 					     "ti,am3352-ecap";
@@ -2225,6 +2243,15 @@
 				status = "disabled";
 			};
 
+			eqep2: eqep@0x48442180 {
+				compatible = "ti,am33xx-eqep";
+				reg = <0x48442180 0x80>;
+				clocks = <&l4_root_clk_div>;
+				clock-names = "fck";
+				interrupt-parent = <&crossbar_mpu>;
+				interrupts = <GIC_SPI 212 IRQ_TYPE_LEVEL_HIGH>;
+			};
+
 			ecap2: ecap@48442100 {
 				compatible = "ti,dra746-ecap",
 					     "ti,am3352-ecap";
-- 
2.7.4

