From e1e5ae2dba4ac65e9ecd5d82df0f622230b5ac9c Mon Sep 17 00:00:00 2001
From: Ezequiel Garcia <ezequiel@vanguardiasur.com.ar>
Date: Mon, 4 Aug 2014 16:18:36 -0300
Subject: [PATCH 1/2] pwm: ehrpwm: Allow to call from interrupt context

Signed-off-by: Ezequiel Garcia <ezequiel@vanguardiasur.com.ar>
---
 drivers/pwm/pwm-tiehrpwm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pwm/pwm-tiehrpwm.c b/drivers/pwm/pwm-tiehrpwm.c
index c791720d27b0..5b08e2d2a488 100644
--- a/drivers/pwm/pwm-tiehrpwm.c
+++ b/drivers/pwm/pwm-tiehrpwm.c
@@ -497,6 +497,7 @@ static int ehrpwm_pwm_probe(struct platform_device *pdev)
 
 	pm_runtime_enable(&pdev->dev);
 	pm_runtime_get_sync(&pdev->dev);
+	pm_runtime_irq_safe(&pdev->dev);
 
 	status = pwmss_submodule_state_change(pdev->dev.parent,
 			PWMSS_EPWMCLK_EN);
-- 
2.15.1

