From 36a06f29b40148a2679ef539d794f2b9cb88895b Mon Sep 17 00:00:00 2001
From: Jason Kridner <jkridner@beagleboard.org>
Date: Sun, 28 Nov 2021 21:50:48 -0500
Subject: [PATCH] Enable eQEP for TDA4VM/J721E

---
 drivers/counter/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/counter/Kconfig b/drivers/counter/Kconfig
index 2de53ab0dd25..453490059c88 100644
--- a/drivers/counter/Kconfig
+++ b/drivers/counter/Kconfig
@@ -51,7 +51,7 @@ config STM32_LPTIMER_CNT
 
 config TI_EQEP
 	tristate "TI eQEP counter driver"
-	depends on (SOC_AM33XX || COMPILE_TEST)
+	depends on (SOC_AM33XX || ARCH_K3 || COMPILE_TEST)
 	select REGMAP_MMIO
 	help
 	  Select this option to enable the Texas Instruments Enhanced Quadrature
-- 
2.30.2

