From cac0e23900225ba61058fa695b17cdc24cd34bd7 Mon Sep 17 00:00:00 2001
From: John Syne <john3909@gmail.com>
Date: Mon, 26 Oct 2015 14:07:20 -0700
Subject: [PATCH 4/4] Update from git.ti.com/rpmsg/rpmsg

Signed-off-by: John Syne <john3909@gmail.com>
---
 drivers/remoteproc/pruss_remoteproc.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/drivers/remoteproc/pruss_remoteproc.c b/drivers/remoteproc/pruss_remoteproc.c
index 3752378..c465dea 100644
--- a/drivers/remoteproc/pruss_remoteproc.c
+++ b/drivers/remoteproc/pruss_remoteproc.c
@@ -313,14 +313,6 @@ static void *pru_i_da_to_va(struct pru_rproc *pru, u32 da, int len)
 {
 	u32 offset;
 
-	/* GNU binutils do not support multiple address spaces. The
-	 * default linker script from the unofficial GNU pru-ld places
-	 * IRAM at an arbitrary high offset, in order to differentiate it
-	 * from DRAM. Hence we need to strip the artificial offset
-	 * from the IRAM address.
-	 */
-	da &= ~0xf0000000u;
-
 	if (len > 0 && da >= pru->iram_da &&
 	    da + len <= pru->iram_da + pru->mem_size[PRU_MEM_IRAM]) {
 		offset = da - pru->iram_da;
-- 
1.9.1

