diff --git a/scripts/prepare-kernel.sh b/scripts/prepare-kernel.sh
index 2d4fb99..9cf25fb 100755
--- a/scripts/prepare-kernel.sh
+++ b/scripts/prepare-kernel.sh
@@ -238,7 +238,7 @@ fi
 # Create an empty output patch file, and initialize the temporary tree.
 if test "x$output_patch" != "x"; then
 
-    temp_tree=$TMPDIR/prepare-kernel-$$
+    temp_tree=/tmp/prepare-kernel-$$
     if ! mkdir $temp_tree; then
 	echo Temporary directory $temp_tree already exists, aborting.
 	exit 1
