From 2b3496fd2d509fca9e77a753e8dcd997a5d523f3 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 31 Aug 2023 21:23:08 -0500
Subject: [PATCH 2/2] hwmon/lm80.c: backport amc80 support

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/hwmon/lm80.c | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/drivers/hwmon/lm80.c b/drivers/hwmon/lm80.c
index 97ab491d2922..b4d7847ec49a 100644
--- a/drivers/hwmon/lm80.c
+++ b/drivers/hwmon/lm80.c
@@ -573,7 +573,11 @@ static int lm80_detect(struct i2c_client *client, struct i2c_board_info *info)
 			return -ENODEV;
 
 		name = "lm96080";
-	} else {
+	}
+	else if(man_id == 0x80 && dev_id == 0x08)  { /* Check for  AMC80 version which is register compatible with lm96080 */
+		name = "amc80"
+	}
+	else {
 		/* Check 6-bit addressing */
 		for (i = 0x2a; i <= 0x3d; i++) {
 			cur = i2c_smbus_read_byte_data(client, i);
@@ -624,6 +628,7 @@ static int lm80_probe(struct i2c_client *client)
 static const struct i2c_device_id lm80_id[] = {
 	{ "lm80", 0 },
 	{ "lm96080", 1 },
+	{ "amc80", 2},
 	{ }
 };
 MODULE_DEVICE_TABLE(i2c, lm80_id);
-- 
2.39.2

