From 587ae21df1cd51e0688091bafcff06b7d66bf01a Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Mon, 24 Sep 2018 15:00:03 -0500
Subject: [PATCH 42/42] fix: include/linux/of.h add linux/slab.h include

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 include/linux/of.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/of.h b/include/linux/of.h
index 9d0e182d4c05..d9db6f7749ee 100644
--- a/include/linux/of.h
+++ b/include/linux/of.h
@@ -26,6 +26,7 @@
 #include <linux/property.h>
 #include <linux/list.h>
 #include <linux/rhashtable.h>
+#include <linux/slab.h>
 
 #include <asm/byteorder.h>
 #include <asm/errno.h>
-- 
2.20.1

