From d70dc3a3f1a15cf52de2f9dc01d53d3c70daa93b Mon Sep 17 00:00:00 2001
From: Ayush Singh <ayush@beagleboard.org>
Date: Tue, 31 Dec 2024 13:53:04 +0530
Subject: [PATCH 9/9] drivers: mspm0: Fix for 6.12

Signed-off-by: Ayush Singh <ayush@beagleboard.org>
---
 drivers/misc/mspm0-i2c.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/misc/mspm0-i2c.c b/drivers/misc/mspm0-i2c.c
index 6a0ce9485124..b8d32437ac6b 100644
--- a/drivers/misc/mspm0-i2c.c
+++ b/drivers/misc/mspm0-i2c.c
@@ -624,9 +624,8 @@ static enum fw_upload_err mspm0_prepare(struct fw_upload *fw_upload,
 
 	if (crc == mspm0->fwl_crc) {
 		bsl_start_application(mspm0);
-		return dev_warn_probe(&mspm0->client->dev,
-				      FW_UPLOAD_ERR_FW_INVALID,
-				      "Skipping reflashing same image");
+		dev_warn(&mspm0->client->dev, "Skipping reflashing same image");
+		return FW_UPLOAD_ERR_FW_INVALID;
 	}
 
 	return bsl_mass_erase(mspm0);
-- 
2.47.1

