From bd202ff3b1afa58ae002a28a8fead48da962ef15 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 22 May 2019 21:03:07 -0500
Subject: [PATCH] auto generated: capes: add dtbs to makefile

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 arch/arm/boot/dts/Makefile | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 95cb911012e6..a47a19ae7b22 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -467,6 +467,16 @@ dtb-$(CONFIG_SOC_AM33XX) += \
 	am335x-base0033.dtb \
 	am335x-bone.dtb \
 	am335x-boneblack.dtb \
+	am335x-pocketbeagle-simplegaming.dtb \
+	am335x-pocketbeagle.dtb \
+	am335x-pocketbone.dtb \
+	am335x-sancloud-bbe-uboot.dtb \
+	am335x-boneblack-uboot.dtb \
+	am335x-bonegreen-modio.dtb \
+	am335x-boneblack-modio.dtb \
+	am57xx-beagle-x15-revb1-ctag.dtb \
+	am335x-vsc8531bbb.dtb \
+	am335x-boneblack-ctag-face.dtb \
 	am335x-boneblack-lcd-ct43.dtb \
 	am335x-boneblack-audio.dtb \
 	am335x-boneblack-bbb-exp-r.dtb \
-- 
2.20.1

