From bd1fabdc37b89c8e60a9ade40fbe0f10a9491f02 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 22 May 2019 21:45:45 -0500
Subject: [PATCH] auto generated: capes: add dtbs to makefile

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 arch/arm/boot/dts/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index e42640e78cef..a13202f670b0 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -467,6 +467,8 @@ dtb-$(CONFIG_SOC_AM33XX) += \
 	am335x-base0033.dtb \
 	am335x-bone.dtb \
 	am335x-boneblack.dtb \
+	am57xx-beagle-x15-revb1-ctag.dtb \
+	am335x-boneblack-ctag-face.dtb \
 	am335x-sancloud-bbe-uboot.dtb \
 	am335x-boneblack-uboot.dtb \
 	am335x-bonegreen-modio.dtb \
-- 
2.20.1

