From 235ea68c5591e2890150cebd7456d119feed8864 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 22 May 2019 21:39:25 -0500
Subject: [PATCH] auto generated: capes: add dtbs to makefile

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 arch/arm/boot/dts/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 9c4b3035b70e..b56fda7d40e9 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -467,6 +467,10 @@ dtb-$(CONFIG_SOC_AM33XX) += \
 	am335x-base0033.dtb \
 	am335x-bone.dtb \
 	am335x-boneblack.dtb \
+	am335x-sancloud-bbe-uboot.dtb \
+	am335x-boneblack-uboot.dtb \
+	am57xx-beagle-x15-revb1-ctag.dtb \
+	am335x-boneblack-ctag-face.dtb \
 	am335x-bonegreen-modio.dtb \
 	am335x-boneblack-modio.dtb \
 	am335x-vsc8531bbb.dtb \
-- 
2.20.1

