From b4be392a4dc07016f6038bd8cd693c7cd65688d9 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 23 May 2019 19:21:02 -0500
Subject: [PATCH] auto generated: capes: add dtbs to makefile

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 arch/arm/boot/dts/Makefile | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 566e702ae701..39c36e7a0fb8 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -622,6 +622,11 @@ dtb-$(CONFIG_SOC_AM33XX) += \
 	am335x-base0033.dtb \
 	am335x-bone.dtb \
 	am335x-boneblack.dtb \
+	am335x-bonegreen-wireless-uboot-univ.dtb \
+	am335x-boneblack-uboot-univ.dtb \
+	am335x-bone-uboot-univ.dtb \
+	am335x-sancloud-bbe-uboot-univ.dtb \
+	am335x-sancloud-bbe-uboot.dtb \
 	am335x-boneblack-wireless-roboticscape.dtb \
 	am335x-boneblack-roboticscape.dtb \
 	am335x-pocketbeagle-techlab.dtb \
@@ -685,6 +690,10 @@ dtb-$(CONFIG_SOC_DRA7XX) += \
 	am57xx-beagle-x15-gssi.dtb \
 	am57xx-beagle-x15-revb1.dtb \
 	am57xx-beagle-x15-revc.dtb \
+	am5729-beagleboneai-roboticscape.dtb \
+	am5729-beagleboneai.dtb \
+	am57xx-evm-reva3.dtb \
+	am57xx-evm.dtb \
 	am57xx-cl-som-am57x.dtb \
 	am57xx-sbc-am57x.dtb \
 	am574x-idk.dtb \
-- 
2.20.1

