From 5990d2a1d0fbb61e21687ab371b2fa41aaf8fe7d Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 8 Mar 2016 12:05:50 -0600
Subject: [PATCH] auto generated: capes: add dtbs to makefile

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 arch/arm/boot/dts/Makefile | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 2c473ed..f9a8c71 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -416,6 +416,25 @@ dtb-$(CONFIG_SOC_AM33XX) += \
 	am335x-base0033.dtb \
 	am335x-bone.dtb \
 	am335x-boneblack.dtb \
+	am335x-bonegreen-ctag-face.dtb \
+	am335x-boneblack-ctag-face.dtb \
+	am335x-sancloud-bbe.dtb \
+	am335x-boneenhanced.dtb \
+	am335x-bonegreen-overlay.dtb \
+	am335x-bonegreen-wireless.dtb \
+	am335x-bonegreen.dtb \
+	am335x-boneblack-wl1835mod.dtb \
+	am335x-boneblack-ttyS2.dtb \
+	am335x-boneblack-spi0.dtb \
+	am335x-boneblack-replicape.dtb \
+	am335x-boneblack-bbbmini.dtb \
+	am335x-boneblack-overlay.dtb \
+	am335x-boneblack-nhdmi-overlay.dtb \
+	am335x-boneblack-hdmi-overlay.dtb \
+	am335x-boneblack-emmc-overlay.dtb \
+	am335x-boneblack-bbb-exp-r.dtb \
+	am335x-boneblack-bbb-exp-c.dtb \
+	am335x-arduino-tre.dtb \
 	am335x-abbbi.dtb \
 	am335x-evm.dtb \
 	am335x-evmsk.dtb \
-- 
2.7.0

