From 31c744bad1de1ba03531f38bdd7187c6c81932d0 Mon Sep 17 00:00:00 2001
From: Takuma Fujiwara <t-fujiwara1@ti.com>
Date: Thu, 25 Apr 2024 19:56:51 -0500
Subject: [PATCH] BeagleY-AI: add pcie_pwr_en in device tree and kernel driver

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/pci/controller/cadence/pci-j721e.c | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/drivers/pci/controller/cadence/pci-j721e.c b/drivers/pci/controller/cadence/pci-j721e.c
index 8137e20b869d..7877f53094a4 100644
--- a/drivers/pci/controller/cadence/pci-j721e.c
+++ b/drivers/pci/controller/cadence/pci-j721e.c
@@ -550,6 +550,15 @@ static int j721e_pcie_probe(struct platform_device *pdev)
 
 	j721e_pcie_config_link_irq(pcie);
 
+	/*
+	 * Enable PCIE_PWR_EN for BeagleY-AI
+	 */
+	gpiod = devm_gpiod_get_optional(dev, "enable", GPIOD_OUT_HIGH);
+	if (IS_ERR(gpiod)) {
+		ret = dev_err_probe(dev, PTR_ERR(gpiod), "Failed to get enable GPIO\n");
+		goto err_get_sync;
+	}
+
 	switch (mode) {
 	case PCI_MODE_RC:
 		gpiod = devm_gpiod_get_optional(dev, "reset", GPIOD_OUT_LOW);
-- 
2.47.2

