From 0ee6a3ec018dc4120851c3a6f98513b2096962f3 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 5 May 2016 12:21:22 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 1199eba..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt16
-- 
2.8.1

