From eae67653c330d92b0af5f7558a2dd392693e209e Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 3 Jan 2017 13:54:27 -0600
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 4b7dca68a5b4..000000000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt49
-- 
2.11.0

