From d4acf5f48e4635719cf1a3548e4214c4576ef3d7 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Fri, 31 Mar 2017 10:28:18 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index d42c0971b041..000000000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt66
-- 
2.11.0

