From 32ec2b46df2df09bcd05359746fdfba8a0d96ae2 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Mon, 12 Dec 2016 16:38:12 -0600
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 41f7b99..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt43
-- 
2.10.2

