From 1a0fbb04009aa2a786ccb631705087114803c1a7 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 16 Jun 2016 08:31:02 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 483ad77..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt19
-- 
2.8.1

