From 6e521f88674d25d0728068df7da3aa787959d011 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 28 Mar 2017 15:04:38 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index d42c0971b041..000000000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt66
-- 
2.11.0

