From eef11a3a5fd2b1f6c317bd4f0c1f82c83c8117fc Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 19 Jan 2017 14:57:47 -0600
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 42c384668389..000000000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt50
-- 
2.11.0

