From 62a06487380ac9f1f96050d71ba9986a780101dc Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 28 Feb 2017 11:06:24 -0600
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 40d81d8e61b6..000000000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt62
-- 
2.11.0

