From 7026765012d242dba0a4b0c863490713ab89e09b Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 15 Sep 2016 10:46:53 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index c5b71f9..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt25
-- 
2.9.3

