From 6992ee427c8ba2d6ec9ab421c703ede0f6051e6c Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 31 May 2016 10:00:13 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 1e584b4..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt17
-- 
2.8.1

