From 9cf091b5b3fcdb97209f8c104d44c588e3746416 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 10 Aug 2016 13:12:38 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index a68b433..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt31
-- 
2.8.1

