From 93da3f250d211f4d1e65ecb1e9a5c2bda897e235 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 13 Apr 2016 10:47:12 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 08b3e75..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt14
-- 
2.8.0.rc3

