From 75a482092107a3f2d4041cd33c1c4373cd75a378 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Fri, 2 Dec 2016 09:29:46 -0600
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 41f7b99..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt43
-- 
2.10.2

