From 7c8ae23e2933b1e57cae0bd342d41f64b3513196 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 15 Feb 2017 14:59:38 -0600
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 8cdeb45db6e2..000000000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt59
-- 
2.11.0

