From 561b0856c0a5a5381c7bd998094534b12de00eea Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Fri, 8 Apr 2016 14:10:53 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 08b3e75..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt14
-- 
2.8.0.rc3

