From d261d2a197bc66e94fd821ee38597ae20407f83b Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 9 Mar 2016 14:41:43 -0600
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 05c35cb..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt11
-- 
2.7.0

