From b1a9a8fb2704f8908f246943764799e0a4e11f3b Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 26 Jan 2017 14:20:04 -0600
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 42c384668389..000000000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt50
-- 
2.11.0

