From 3ddfcabcea146356a91902c97c892df6b1619cf9 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Fri, 15 Jul 2016 08:39:32 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index a68b433..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt31
-- 
2.8.1

