From a7ab2493333277f87e914535c12b0b3ba13f9a91 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Mon, 24 Oct 2016 09:16:27 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index e1d8362..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt33
-- 
2.9.3

