From f15ed45f0b4648f4eb5be4010f590055d125b19d Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Fri, 10 Feb 2017 13:51:00 -0600
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index f9df2cf089cf..000000000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt58
-- 
2.11.0

