From 62ab205d590dcda18b045be3d82f77ad654238df Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Fri, 15 Apr 2016 13:53:44 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 1199eba..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt16
-- 
2.8.0.rc3

