From a1eb228d7d035b07113e5db9dcadb790699b08a9 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 12 May 2016 12:24:03 -0500
Subject: [PATCH] merge: CONFIG_PREEMPT_RT Patch Set

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 localversion-rt | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
deleted file mode 100644
index 9a218ca..0000000
--- a/localversion-rt
+++ /dev/null
@@ -1 +0,0 @@
--rt23
-- 
2.8.1

