From 3c497afb3f3a13f3c868de1953eb242e51aa383f Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 4 Oct 2018 10:39:48 -0500
Subject: [PATCH 2/2] unstage typec

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/staging/Kconfig  | 2 --
 drivers/staging/Makefile | 1 -
 2 files changed, 3 deletions(-)

diff --git a/drivers/staging/Kconfig b/drivers/staging/Kconfig
index 554683912cff..1e3d5d0bc2de 100644
--- a/drivers/staging/Kconfig
+++ b/drivers/staging/Kconfig
@@ -112,8 +112,6 @@ source "drivers/staging/vc04_services/Kconfig"
 
 source "drivers/staging/ccree/Kconfig"
 
-source "drivers/staging/typec/Kconfig"
-
 source "drivers/staging/vboxvideo/Kconfig"
 
 source "drivers/staging/pi433/Kconfig"
diff --git a/drivers/staging/Makefile b/drivers/staging/Makefile
index 6e536020029a..ba69c5294b01 100644
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -2,7 +2,6 @@
 # Makefile for staging directory
 
 obj-y				+= media/
-obj-y				+= typec/
 obj-$(CONFIG_IRDA)		+= irda/net/
 obj-$(CONFIG_IRDA)		+= irda/drivers/
 obj-$(CONFIG_PRISM2_USB)	+= wlan-ng/
-- 
2.19.0

